src/github/createPRViewProviderNew.ts

```diff
@@ -623,7 +624,10 @@ export class CreatePullRequestViewProviderNew extends WebviewViewBase implements
 			return newLabels.map(label => {
 				return {
 					label: label.name,
-					picked: labels.some(existingLabel => existingLabel.name === label.name)
+					picked: labels.some(existingLabel => existingLabel.name === label.name),
+					iconPath: DataUri.asImageDataURI(Buffer.from(`<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is `Buffer` polyfilled in web?

Reply:
Agh, this needs to be fixed. Thank you for catching!